crates/runtime/src/http/v1.rs

```diff
@@ -1040,9 +1077,20 @@ pub(crate) mod nsql {
         match result {
             Ok(Some(model_sql_query)) => {
                 let cleaned_query = clean_model_based_sql(&model_sql_query);
+                q_trace = q_trace.sql(cleaned_query.clone());
                 tracing::trace!("Running query:\n{cleaned_query}");
-                match readable_df.ctx.sql(&cleaned_query).await {
-                    Ok(result) => dataframe_to_response(result).await,
+
+                q_trace = q_trace.start_time(SystemTime::now());
+                let result = readable_df.ctx.sql(&cleaned_query).await;
+                q_trace = q_trace.end_time(SystemTime::now());
```

Comment:
it this supposed to trace the dataframe with sql time? or intended for the dataframe_tro_response time.

Reply:
ahh it should track both 1. `readable_df.ctx.sql` and 2.  `dataframe_to_response`. Essentially to give as close as an analogy to the standard `POST v1/sql` endpoint.